core/restclient.class.inc.php

```diff
@@ -204,4 +206,40 @@ public static function GetFullSynchroDataSource(&$aSource, $iSourceId)
 
 		return $bResult;
 	}
+	
+	/**
+	 * Check if the given module is installed in iTop
+	 * @param string $sName Name of the module to be found
+	 * @param bool $bRequired Whether to throw exceptions when module not found
+	 * @return bool True when the given module is installed, false otherwise
+	 * @throws Exception When the module is required but could not be found
+	 */
+	public function CheckModuleInstallation(string $sName, bool $bRequired = false): bool
+	{
+		try {
+			if (!isset($this->sLastInstallDate)) {
+				$aDatamodelResults = static::Get('ModuleInstallation', ['name' => 'datamodel'], 'installed', 1);
+				if ($aDatamodelResults['code'] != 0 || empty($aDatamodelResults['objects'])){
```

Comment:
```suggestion
				if ($aDatamodelResults['code'] != 0 || count($aDatamodelResults['objects']) === 0){
```

We tend to avoid using `empty()`

Reply:
I'll revert this since this will give warnings when no objects returned (datatype is `null`). See https://github.com/Combodo/iTop/pull/231

Reply:
Good catch